Modules/Loadable/Segmentations/EditorEffects/qSlicerSegmentEditorPaintEffect.cxx

```diff
@@ -234,8 +235,7 @@ qSlicerSegmentEditorPaintEffectPrivate::qSlicerSegmentEditorPaintEffectPrivate(q
   , ActiveViewWidget(nullptr)
   , PaintOptionsFrame(nullptr)
   , BrushDiameterFrame(nullptr)
-  , BrushDiameterSpinBox(nullptr)
-  , BrushDiameterSlider(nullptr)
+  , BrushDiameterSliderWidget(nullptr)
```

Comment:
Add initialization of `BrushDiameterSizeFrame`.
To avoid all these initializations here (and risk missing some of them), it would be probably better to initialize all in the header file.